rueidis.go

```diff
@@ -127,6 +127,8 @@ type ClientOption struct {
 
 	// ShuffleInit is a handy flag that shuffles the InitAddress after passing to the NewClient() if it is true
 	ShuffleInit bool
+	// NoTouch controls whether commands alter LRU/LFU stats
+	NoTouch bool
```

Comment:
Hi @SoulPancake, Thanks!

I think `ClientNoTouch` would be better, aligning with the current `ClientName` option.